### PR TITLE
feat: Add es samples for 11.2.2

### PR DESCRIPTION
**Pull request by varada-sunanda-ibm**
Samples update for Event Streams Release 11.2.2/3.2.2

### CODE REVIEW

**Review by ajborley — APPROVED** [review]
LGTM - thanks

File: event-streams/README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
-# Event Streams operator resources
-View and download the samples for Event Streams.
\ No newline at end of file
+#  Event Streams Samples
```
Comment (gabortpubs) [inline event-streams/README.md:1]:
```suggestion
#  Event Streams samples
```

File: event-streams/README.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
...
+This directory contains the below folders/files that can be used for different features provided by Event Streams :
```
Comment (gabortpubs) [inline event-streams/README.md:2]:
```suggestion
The following folders and files contain sample configurations you can use to set up Event Streams:
```

File: event-streams/README.md [hunk 3]
```diff
@@ -1,2 +1,7 @@
...
+- cr-examples : Examples of various custom resources managed by the Event Streams operator.
```
Comment (gabortpubs) [inline event-streams/README.md:3]:
```suggestion
- cr-examples: Sample configurations you can base your Event Streams deployment on.
```

File: event-streams/README.md [hunk 4]
```diff
@@ -1,2 +1,7 @@
...
+- grafana-dashboards : Examples to configure cluster health using prometheus and grafana.
```
Comment (gabortpubs) [inline event-streams/README.md:4]:
```suggestion
- grafana-dashboards: Sample Grafana dashboard configuration files to help you set up cluster health monitoring with Prometheus and Grafana.
```

File: event-streams/README.md [hunk 5]
```diff
@@ -1,2 +1,7 @@
...
+- kibana-dashboards : Json sample to configure kibana dashboards.
```
Comment (gabortpubs) [inline event-streams/README.md:5]:
```suggestion
- kibana-dashboards: Sample JSON configuration files to help you set up Kibana dashboards for Event Streams.
```
(although there's only one)

File: event-streams/README.md [hunk 6]
```diff
@@ -1,2 +1,7 @@
...
+- metrics-config : Example configuration for Kafka / Zookeeper metrics collection.
```
Comment (gabortpubs) [inline event-streams/README.md:6]:
```suggestion
- metrics-config: Sample configuration to collect Kafka and Zookeeper metrics. 
```

File: event-streams/README.md [hunk 7]
```diff
@@ -1,2 +1,7 @@
...
+- eventstreams-scc.yaml : The minimal security context constraints required for EventStreams and its components to function.
```
Comment (gabortpubs) [inline event-streams/README.md:7]:
```suggestion
- eventstreams-scc.yaml: The minimal security context constraints (SCCs) required for Event Streams and its components to function.
```

File: event-streams/cr-examples/README.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
+# Custome Resource Example
```
Comment (gabortpubs) [inline event-streams/cr-examples/README.md:1]:
```suggestion
# Configuration samples
```

File: event-streams/cr-examples/README.md [hunk 9]
```diff
@@ -0,0 +1,7 @@
...
+This directory contains different custom resource examples to configure
```
Comment (gabortpubs) [inline event-streams/cr-examples/README.md:2]:
```suggestion
This directory contains sample configurations for the following custom resources:
```

File: event-streams/cr-examples/README.md [hunk 10]
```diff
@@ -0,0 +1,7 @@
...
+- EventStreams instance on kubernetes and openshift platforms
```
Comment (gabortpubs) [inline event-streams/cr-examples/README.md:3]:
```suggestion
- `EventStreams` for configuring an instance of Event Streams on Kubernetes and OpenShift platforms.
```

File: event-streams/cr-examples/README.md [hunk 11]
```diff
@@ -0,0 +1,7 @@
...
+- KafkaConnect custom resource configurations for different environments
```
Comment (gabortpubs) [inline event-streams/cr-examples/README.md:4]:
```suggestion
- `KafkaConnect` for configuring an instance of a Kafka Connect node hosted by Event Streams.
```

File: event-streams/cr-examples/README.md [hunk 12]
```diff
@@ -0,0 +1,7 @@
...
+- EventStreamsGeoReplicator custom resources for data replication
```
Comment (gabortpubs) [inline event-streams/cr-examples/README.md:5]:
```suggestion
- `EventStreamsGeoReplicator` to enable data replication by setting up geo-replication.
```

File: event-streams/cr-examples/README.md [hunk 13]
```diff
@@ -0,0 +1,7 @@
...
+- KafkaMirrorMaker2 custom resources for data replication
```
Comment (gabortpubs) [inline event-streams/cr-examples/README.md:6]:
```suggestion
- `KafkaMirrorMaker2` to enable data replication by setting up geo-replication.
```

File: event-streams/cr-examples/README.md [hunk 14]
```diff
@@ -0,0 +1,7 @@
...
+- KafkaUser custom resources with various permissions and security configurations (scram/tls)
```
Comment (gabortpubs) [inline event-streams/cr-examples/README.md:7]:
```suggestion
- `KafkaUser` to manage access by setting permissions and security configurations.
```

File: event-streams/cr-examples/eventstreams-georeplicator/README.md [hunk 15]
```diff
@@ -0,0 +1,2 @@
+# Sample for Geo-Replication 
+This folder contains an example for creating an EventStreamsGeoReplicator instance for event streams.
```
Comment (gabortpubs) [inline event-streams/cr-examples/eventstreams-georeplicator/README.md:2]:
```suggestion
This folder contains an example for creating an `EventStreamsGeoReplicator` instance for Event Streams to enable data replication.
```

File: event-streams/cr-examples/eventstreams/README.md [hunk 16]
```diff
@@ -0,0 +1,2 @@
+# Event Streams Samples
```
Comment (gabortpubs) [inline event-streams/cr-examples/eventstreams/README.md:1]:
```suggestion
# Event Streams samples
```

File: event-streams/cr-examples/eventstreams/README.md [hunk 17]
```diff
@@ -0,0 +1,2 @@
...
+This directory provides sample configuration for installing Event Streams on Kubernetes and OpenShift Container Platforms.
```
Comment (gabortpubs) [inline event-streams/cr-examples/eventstreams/README.md:2]:
```suggestion
This directory provides sample configurations for installing Event Streams on Kubernetes and OpenShift Container Platforms.
```

File: event-streams/cr-examples/eventstreams/kubernetes/README.md [hunk 18]
```diff
@@ -0,0 +1,8 @@
+# Event Streams samples for Kubernetes Platform 
+
+This directory provides sample configuration for installing EventStreams on a Kubernetes Platform.
```
Comment (gabortpubs) [inline event-streams/cr-examples/eventstreams/kubernetes/README.md:3]:
```suggestion
This directory provides sample configurations for installing Event Streams on Kubernetes platforms.
```

File: event-streams/cr-examples/eventstreams/openshift/README.md [hunk 19]
```diff
@@ -0,0 +1,8 @@
+# Event Streams samples for OpenShift Container Platform
```
Comment (gabortpubs) [inline event-streams/cr-examples/eventstreams/openshift/README.md:1]:
```suggestion
# Event Streams samples for the Red Hat OpenShift Container Platform
```

File: event-streams/cr-examples/eventstreams/openshift/README.md [hunk 20]
```diff
@@ -0,0 +1,8 @@
...
+
+This directory provides sample configuration for installing Event Streams on OpenShift Container Platform .
```
Comment (gabortpubs) [inline event-streams/cr-examples/eventstreams/openshift/README.md:3]:
```suggestion
This directory provides sample configurations for installing Event Streams on the OpenShift Container Platform.
```

File: event-streams/cr-examples/kafka-connect/README.md [hunk 21]
```diff
@@ -0,0 +1,5 @@
+# Setting up and running connectors
```
Comment (gabortpubs) [inline event-streams/cr-examples/kafka-connect/README.md:1]:
```suggestion
# Kafka Connect samples
```

File: event-streams/cr-examples/kafka-mirror-maker-2/README.md [hunk 22]
```diff
@@ -0,0 +1,2 @@
+# Configuring Kafka cluster to use  MirrorMaker 2
```
Comment (gabortpubs) [inline event-streams/cr-examples/kafka-mirror-maker-2/README.md:1]:
```suggestion
# Sample configuration for MirrorMaker2
```

File: event-streams/cr-examples/kafka-user/README.md [hunk 23]
```diff
@@ -0,0 +1,2 @@
+# KafkaUser Examples
```
Comment (gabortpubs) [inline event-streams/cr-examples/kafka-user/README.md:1]:
```suggestion
# KafkaUser samples
```

File: event-streams/grafana-dashboards/README.md [hunk 24]
```diff
@@ -0,0 +1,7 @@
+# Grafana-Dashboards Samples
```
Comment (gabortpubs) [inline event-streams/grafana-dashboards/README.md:1]:
```suggestion
# Grafana dashboards samples
```

File: event-streams/kibana-dashboards/README.md [hunk 25]
```diff
@@ -0,0 +1,2 @@
+# Examples for Kibana dashboards for Event Streams
```
Comment (gabortpubs) [inline event-streams/kibana-dashboards/README.md:1]:
```suggestion
# Sample Kibana dashboard for Event Streams 
```

File: event-streams/metrics-config/README.md [hunk 26]
```diff
@@ -0,0 +1,2 @@
+## Event Streams metrics examples
```
Comment (gabortpubs) [inline event-streams/metrics-config/README.md:1]:
```suggestion
## Event Streams metrics samples
```

File: event-streams/metrics-config/README.md [hunk 27]
```diff
@@ -0,0 +1,2 @@
...
+This folder contains configuration to fetch metrics from the panels in the dashboard.
```
Comment (gabortpubs) [inline event-streams/metrics-config/README.md:2]:
```suggestion
This folder contains sample configurations to collect Kafka and Zookeeper metrics.
```

File: event-streams/grafana-dashboards/README.md [hunk 28]
```diff
@@ -0,0 +1,7 @@
...
+This directory contains files for configuring Event Streams cluster health with Prometheus and Grafana
```
Comment (gabortpubs) [inline event-streams/grafana-dashboards/README.md:2]:
```suggestion
This directory contains sample configuration files to help you set up cluster health monitoring with Prometheus and Grafana.
```

File: event-streams/grafana-dashboards/README.md [hunk 29]
```diff
@@ -0,0 +1,7 @@
...
+- [ibm-eventstreams-georeplication-dashboard.yaml](./ibm-eventstreams-georeplication-dashboard.yaml) : Example dashboard for monitoring georeplication/mirrormaker2 workers.
```
Comment (gabortpubs) [inline event-streams/grafana-dashboards/README.md:3]:
```suggestion
- [ibm-eventstreams-georeplication-dashboard.yaml](./ibm-eventstreams-georeplication-dashboard.yaml): Sample dashboard for monitoring geo-replication and MirrorMaker2 workers.
```

File: event-streams/grafana-dashboards/README.md [hunk 30]
```diff
@@ -0,0 +1,7 @@
...
+- [ibm-eventstreams-kafka.json](./ibm-eventstreams-kafka.json) : Example dashboard for monitoring Kafka cluster health.
```
Comment (gabortpubs) [inline event-streams/grafana-dashboards/README.md:4]:
```suggestion
- [ibm-eventstreams-kafka.json](./ibm-eventstreams-kafka.json): Sample dashboard for monitoring Kafka cluster health.
```

File: event-streams/grafana-dashboards/README.md [hunk 31]
```diff
@@ -0,0 +1,7 @@
...
+- [ibm-eventstreams-kafka-health-dashboard.yaml](ibm-eventstreams-kafka-health-dashboard.yaml) : Example Grafana JSON dashboard for monitoring Kafka cluster.
```
Comment (gabortpubs) [inline event-streams/grafana-dashboards/README.md:5]:
```suggestion
- [ibm-eventstreams-kafka-health-dashboard.yaml](ibm-eventstreams-kafka-health-dashboard.yaml): Sample Grafana JSON dashboard for monitoring your Kafka cluster.
```

File: event-streams/grafana-dashboards/README.md [hunk 32]
```diff
@@ -0,0 +1,7 @@
...
+- [ibm-eventstreams-operator-dashboard.yaml](./ibm-eventstreams-operator-dashboard.yaml) : Example dashboard for monitoring Event Streams operator.
```
Comment (gabortpubs) [inline event-streams/grafana-dashboards/README.md:6]:
```suggestion
- [ibm-eventstreams-operator-dashboard.yaml](./ibm-eventstreams-operator-dashboard.yaml): Sample dashboard for monitoring the Event Streams operator.
```

File: event-streams/grafana-dashboards/README.md [hunk 33]
```diff
@@ -0,0 +1,7 @@
...
+- [ibm-eventstreams-performance-dashboard.yaml](./ibm-eventstreams-performance-dashboard.yaml) : Example dashboard for monitoring performance of Event Streams pods.
```
Comment (gabortpubs) [inline event-streams/grafana-dashboards/README.md:7]:
```suggestion
- [ibm-eventstreams-performance-dashboard.yaml](./ibm-eventstreams-performance-dashboard.yaml): Sample dashboard for monitoring the performance of Event Streams pods.
```

File: event-streams/grafana-dashboards/README.md (hunk 28 above)
Comment (gabortpubs) [reply event-streams/grafana-dashboards/README.md:2]: Is there a reason we provide an explanation and links into the samples within the folder here, but we don't do it for others the same way (for example, `kafka-connect` or `kafka-user` dirs, or any other which has more than one sample).

**Review by gabortpubs — COMMENTED** [review]
Left suggestions and a question.

File: event-streams/grafana-dashboards/README.md (hunk 28 above)
Comment (varada-sunanda-ibm) [reply event-streams/grafana-dashboards/README.md]: @gabortpubs I believe we don't provide much information about the grafana configuration files in our official docs but for the rest of them we do have and  For `kafka-user` dirs there are a lot of files and I dint want to overdo a lot of information into the Readme.

But for a matter of consistency, we can remove the explanation and the links to the configuration file in the grafana directory.

File: event-streams/kibana-dashboards/README.md [hunk 34]
```diff
@@ -0,0 +1,2 @@
+# Sample Kibana dashboard for Event Streams 
+This folder contains a sample configuration file to help you set up Kibana dashboards for Event Streams
```
Comment (gabortpubs) [inline event-streams/kibana-dashboards/README.md:2]:
```suggestion
This folder contains a sample configuration file to help you set up Kibana dashboards for Event Streams.
```

**Review by gabortpubs — APPROVED** [review]

File: event-streams/cr-examples/kafka-mirror-maker-2/README.md [hunk 35]
```diff
@@ -0,0 +1,2 @@
+# Sample configuration for MirrorMaker2
+This folder provides a sample custom resources that can be used to configure `KafkaMirrorMaker2` to help with data migration.
```
Comment (ajborley) [inline event-streams/cr-examples/kafka-mirror-maker-2/README.md:2]:
```suggestion
This folder provides sample custom resources that can be used to configure `KafkaMirrorMaker2` to help with data migration.
```

**Review by ajborley — APPROVED** [review]
Approved with a couple of suggestions- thanks!

**Pull request merged**